### ISSUE TITLE
[Bugs] Lunox Depencency missing when using node v17.4.0

### ISSUE THREAD

**Issue by axmad386** [kodepandai/lunox#18]
When we start lunox in dev mode `pnpm dev` lunox is crashing with node v17.4.0, but it's fine in node v16.
```bash
compiling ts file...

node:internal/process/esm_loader:94
    internalBinding('errors').triggerUncaughtException(
                              ^

Error [ERR_MODULE_NOT_FOUND]: Cannot find package '@kodepandai/flydrive' imported from /home/dmuggle/GIT/cobalunox/node_modules/lunox/dist/index.js
Did you mean to import @kodepandai+flydrive@2.0.0/node_modules/@kodepandai/flydrive/build/index.js?
    at new NodeError (node:internal/errors:371:5)
    at packageResolve (node:internal/modules/esm/resolve:892:9)
    at moduleResolve (node:internal/modules/esm/resolve:938:18)
    at defaultResolve (node:internal/modules/esm/resolve:1041:11)
    at ESMLoader.resolve (node:internal/modules/esm/loader:530:30)
    at ESMLoader.getModuleJob (node:internal/modules/esm/loader:251:18)
    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:79:40)
    at link (node:internal/modules/esm/module_job:78:36) {
  code: 'ERR_MODULE_NOT_FOUND'
}

Node.js v17.4.0

rollup v2.65.0
```
I still investigate this issue. For now, please using node v16.x.x

**Comment by axmad386** [comment]
Please refer to this issue for tracking https://github.com/pnpm/pnpm/issues/4364

**Comment by axmad386** [comment]
fixed in 7c6236112a92ad13037bfce36fc35711816322ec. Tested using node v17.4.0
